File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = {0,0};
        int i=0;
      while(i < num_list.length) {
          if(num_list[i]%2==0)
              answer[0] ++;
          else
              answer[1] ++;
          i++;
      }  
        return answer;
    }
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    let answer = [0,0];
    for(let i=1; i<=num_list.length; i++) {
        let index = num_list[i-1]%2;
        console.log(answer);
        answer[index]++;
    }
    
    return answer;
}